tools/cldr-code/src/main/java/org/unicode/cldr/tool/Main.java

```diff
@@ -38,6 +40,17 @@ public static void main(String args[]) throws Throwable {
             System.arraycopy(args, 1, args2, 0, args2.length);
             Class<?> c = null;
 
+            try {
```

Comment:
quibble: better to make a subroutine; main is too long already

Reply:
makes sense